packages/logs/src/boot/logsPublicApi.ts

```diff
@@ -163,9 +164,7 @@ export function makeLogsPublicApi(startLogsImpl: StartLogs) {
 
   function canInitLogs(initConfiguration: LogsInitConfiguration) {
     if (isAlreadyInitialized) {
-      if (!initConfiguration.silentMultipleInit) {
-        display.error('DD_LOGS is already initialized.')
-      }
+      displayAlreadyInitializedError('DD_LOGS', initConfiguration)
```

Comment:
`canInitSdk` has a state associated to it (`isAlreadyInitialized`), so it would be more complex than just a plain function and I don't think it's worth it. WDYT?

Reply:
Yes, right. You can leave it this way. No problem